HW_2/home_work_2.py

```diff
@@ -0,0 +1,59 @@
+import random
+import string
+import csv
+import pandas as pd
+from openpyxl import load_workbook, Workbook
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def hello_world():
+    return "<p>Hello, World!!</p>"
+
+
+@app.route("/password")
+def generate_password():
+    password_lenght = random.randint(10,20)
+    pass_symbols = string.digits + string.ascii_lowercase + string.ascii_lowercase + "!#$%&/()>=?_+-"
+    password = ''.join(random.choice(pass_symbols) for _ in range(password_lenght))
```

Comment:
your password must contain at least one of:
     ascii_lowercase
     ascii_uppercase
     int
     special symbols
     
 In this implementation sometimes you don't have all of them.